e2e/integration/ssitests.js

```diff
@@ -12,45 +12,61 @@ context( 'Test for SSI', () => {
     } );
 
     it( 'Test for Notifications section', () => {
-        cy.get( '.op-nav-wrapper' ).within( () => {
-            cy.contains( 'Notifications' ).click();
-        } );
-        cy.get( '#op-menu-drawer' ).should( 'be.visible' ).within( () => {
-            cy.get( '.op-menu-drawer__title' ).should( 'contain.text', 'All Notifications ' );
+        cy.get( 'slot[name="opc-nav-btn"] > button:first-child' ).click({force:true}).then( () => {
+            cy.wait( 2000 )
+            cy.get( '.opc-notification-drawer__header-title',{force:true} ).invoke( 'text' ).then( ( text ) => {
```

Comment:
@mudit94 I think targeting id's would be better, because in future the class name might change but the id's tend to remain the same

Reply:
Hi @hybridx , ids are not available for most of the elements